modules/gdscript/gdscript.cpp

```diff
@@ -648,10 +643,7 @@ Variant GDScript::call(const StringName &p_method, const Variant **p_args, int p
 		Map<StringName, GDScriptFunction *>::Element *E = top->member_functions.find(p_method);
 		if (E) {
 
-			if (!E->get()->is_static()) {
-				ERR_EXPLAIN("Can't call non-static function: '" + String(p_method) + "' in script.");
-				ERR_FAIL_V(Variant());
-			}
+			ERR_FAIL_COND_V_MSG(!E->get()->is_static(), Variant(), "Can't call non-static function: '" + String(p_method) + "' in script.");
```

Comment:
Same here with the unnecessary colon IMO (but that's veeery nitpicky, and feel free to disagree).

Reply:
I'll just stick to what's common, so I'll remove it :)